### ISSUE TITLE
Avoid using named import from version.json

### ISSUE THREAD

**Issue by sxzz** [travist/jsencrypt#196]
Webpack 5 warning:
```
Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)
```

Consider using the default export please.

https://github.com/travist/jsencrypt/blob/9aaa655b4d944a1e6177b9a9f3b4abff11157cdf/src/JSEncrypt.ts#L3

**Comment by hxuena** [comment]
I ran into the same question. "Uncaught Error: Cannot find module './version.json' at webpackMissingModule (version.json:1)"

**Comment by travist** [comment]
Fixed in 3.0.3

**Comment by sxzz** [comment]
Thanks.
